src/usr/local/www/diag_packet_capture.php

```diff
@@ -442,6 +443,10 @@ function fixup_host($value, $position) {
 		null,
 		'fa-stop-circle'
 	))->addClass('btn-warning');
+	$section->addInput(new Form_StaticText(
```

Comment:
This should be wrapped in a `file_exists()` test. When trying this out, I received an error:

```
Warning: filemtime(): stat failed for /root/packetcapture.start in /usr/local/www/diag_packet_capture.php on line 480 
```

This will never display after clicking `Start` because the file is touched after this block of code is run. It does show if the page is refreshed after.

Reply:
fixed